### PR TITLE
Fix boolean inputs

### LINKED ISSUE #6: The options don't work
https://github.com/sindresorhus/refined-github/blob/18387e73251a024b26bb7d58f77d33bcae15b4cd/.github/workflows/deployment.yml#L27

```yml
      - uses: notlmn/release-with-changelog@v2
        with:
          include-range: false
          token: ${{ secrets.GITHUB_TOKEN }}
```

Generated:

```md
- [`18387e73`](https://github.com/sindresorhus/refined-github/commits/18387e73251a024b26bb7d58f77d33bcae15b4cd) Add `stop-pjax-loading-with-esc` feature (#3465)
- [`d8c1c195`](https://github.com/sindresorhus/refined-github/commits/d8c1c195cfdff5031fda6dc688ba36828e9d7d49) Remove `deinit` in `remove-label-faster` (#3538)
- [`2dc5f263`](https://github.com/sindresorhus/refined-github/commits/2dc5f263b97730b82ceee8cb64edc5da76fe5893) Purge expired cache items
- [`760abcfc`](https://github.com/sindresorhus/refined-github/commits/760abcfc4b2db013340241f87175a7b91ac11403) Correctly dim `bot` label in `dim-bots` (#3532)
- [`c254e559`](https://github.com/sindresorhus/refined-github/commits/c254e5596b852fa4b73177481bebc182f321b5d0) Ensure `batch-open-conversations` always adds every checkbox

[`20.9.4.1817..20.9.10`](https://github.com/sindresorhus/refined-github/compare/20.9.4.1817..20.9.10)
```

`include-range` is set to `false`, but the link is still included

### PR DESCRIPTION
**Pull request by notlmn**
Closes #6

GitHub Actions does not support boolean inputs in yaml files yet.
See: https://github.com/actions/toolkit/issues/361

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -9,8 +9,10 @@ async function run() {
 
 		const header = core.getInput('header');
 		const footer = core.getInput('footer');
-		const includeHash = core.getInput('include-hash');
-		const includeRange = core.getInput('include-range');
+
+		// @TODO: Fix boolean checks when https://github.com/actions/toolkit/issues/361 gets resolved
+		const includeHash = core.getInput('include-hash').toString() === 'true';
+		const includeRange = core.getInput('include-range').toString() === 'true';
```
Comment (fregante) [inline index.js]: Why toString? Aren’t they already strings?

Comment (notlmn) [reply index.js]: Just in case they decide to make booleans work, or return `null` or `undefined` if the inputs are not passed.

Comment (notlmn) [reply index.js]: Meh, we're bundling code anyway, will drop it.

**Comment by fregante** [comment]
LGTM

**Comment by notlmn** [comment]
Re-tagged and re-released https://github.com/notlmn/release-with-changelog/releases/tag/v2 👏

**Pull request merged**